util/dvsim/Testplan.py

```diff
@@ -578,6 +582,15 @@ def _process_testpoint(testpoint, totals):
                     totals["N.A."].test_results[0].passing += tr.passing
                     totals["N.A."].test_results[0].total += tr.total
 
+                # Assign the max job runtime to each milestone total.
+                if tr.job_runtime > totals[ms].test_results[0].job_runtime:
+                    totals[ms].test_results[0].job_runtime = tr.job_runtime
+                    totals[ms].test_results[0].simulated_time = tr.simulated_time
+                    # Assign the max job runtime to the sim total result.
+                    if tr.job_runtime > totals["N.A."].test_results[0].job_runtime:
+                        totals["N.A."].test_results[0].job_runtime = tr.job_runtime
+                        totals["N.A."].test_results[0].simulated_time = tr.simulated_time
```

Comment:
For V1/2/3 / Grand Total rows, I think we do not need to compute these parameters. Instead, we could "highlight" the cell containing the worst 3 runtimes in shades of red (or something like that).

Reply:
Okay sounds good! I will just use `N.A.` for totals :)

Reply:
Thanks Matute, let me get through the comments as much as I can, and I will file an issue on Monday with all the remaining ideas! Sorry about that. Hope did not block you for your coming work. @matutem @sriyerg